app/components/slack-context-menu.js

```diff
@@ -138,7 +149,7 @@
 	}
 
 	module.exports = {
-		handleRightClick: function (window, target, x, y) {
+		handleRightClick: function (window, target, x, y, teamUrl) {
```

Comment:
These parameters are getting a bit unwieldy. Maybe we should convert them into an object for easier passing?

```js
// params = {window, target, x, y, teamUrl}
function (params) {
```